lib/engines/content_object_store/app/lib/content_object_store/publishable.rb

```diff
@@ -30,6 +30,17 @@ def publish_with_rollback(schema:, title:, details:)
       end
     end
 
+    def schedule_with_rollback
+      raise ArgumentError, "Local database changes not given" unless block_given?
+
+      ActiveRecord::Base.transaction do
+        content_block_edition = yield
+
+        content_block_edition.public_send(:schedule!)
```

Comment:
I can't find where `.schedule!` comes from? Where did it come from and what does it do?

Reply:
ah yep this is copy/paste cruft from Whitehall, not sure why they do `public_send`... will re-order commits and just call `.schedule!`